### PR TITLE
copy content of finder page

### PR DESCRIPTION
**Pull request by Afrasyabafzal**

### CODE REVIEW

**Comment by MZO9400** [comment]
@Afrasyabafzal merge conflicts

File: src/components/CallToAction.jsx [hunk 1]
```diff
@@ -14,12 +14,10 @@ export function CallToAction() {
       <Container className="relative">
         <div className="mx-auto max-w-md sm:text-center">
           <h2 className="text-3xl font-medium tracking-tight text-white sm:text-4xl">
-            Get your first tips today
+            Start Deploying Your Delivery Fleet
```
Comment (MZO9400) [inline src/components/CallToAction.jsx:17]: Do not modify components

File: src/components/CallToAction.jsx [hunk 2]
```diff
@@ -14,12 +14,10 @@ export function CallToAction() {
...
         <div className="mx-auto max-w-md sm:text-center">
           <h2 className="text-3xl font-medium tracking-tight text-white sm:text-4xl">
-            Get your first tips today
+            Start Deploying Your Delivery Fleet 
           </h2>
           <p className="mt-4 text-lg text-gray-300">
-            It takes 30 seconds to sign up. Download the app and create an
-            account today and we’ll send you a tip guaranteed to double your
-            first investment.
+            You may begin adding your delivery drivers as soon as you select any plan that includes GAPP Finder Basic or GAPP Finder Plus.
```
Comment (MZO9400) [inline src/components/CallToAction.jsx:20]: +1

File: src/components/HeroSection.jsx [hunk 3]
```diff
@@ -64,12 +64,11 @@ export default function HeroSection() {
         <main className="mt-16 mx-auto max-w-7xl px-4 sm:mt-24">
           <div className="text-center">
             <h1 className="text-4xl tracking-tight font-bold text-gray-900 sm:text-5xl md:text-6xl">
-              <span className="block xl:inline">Data to enrich your</span>{' '}
-              <span className="block text-accent xl:inline">online business</span>
+              <span className="block xl:inline">GAPP Finder – </span>{' '}
```
Comment (MZO9400) [inline src/components/HeroSection.jsx:67]: +1

File: src/components/HeroSection.jsx [hunk 4]
```diff
@@ -64,12 +64,11 @@ export default function HeroSection() {
...
+              <span className="block text-accent xl:inline">Driving The Delivery Economy </span>
             </h1>
             <p className="mt-3 max-w-md mx-auto text-base text-gray-500 sm:text-lg md:mt-5 md:text-xl md:max-w-3xl">
-              Anim aute id magna aliqua ad ad non deserunt sunt. Qui irure qui lorem cupidatat commodo. Elit sunt amet
-              fugiat veniam occaecat fugiat aliqua.
+              Improve productivity, increase revenues, and provide a great client experience at scale by centralizing management of all your delivery activities on a single platform.
```
Comment (MZO9400) [inline src/components/HeroSection.jsx:71]: +1

**Review by MZO9400 — CHANGES_REQUESTED** [review]
All of the changes need to be redone, with prop drilling from pages to components.

Keep components generic

**Comment by Afrasyabafzal** [comment]
@MZO9400 Done Have a look

**Review by MZO9400 — APPROVED** [review]

**Review by MZO9400 — APPROVED** [review]

**Pull request merged**